app/serializers/user_serializer.rb

```diff
@@ -25,7 +25,7 @@
 #  account_id             :integer          not null
 #
 
-class User < ActiveModel::Serializer
+class UserSerializer < ActiveModel::Serializer
```

Comment:
Gah, so many typos in my rush-code.  Thanks for fixing!

Reply:
Actually I think you uncovered / fixed this elsewhere as we were rushing to deploy during the Hackathon and it just never got merged.